src/tokens/TokenBalanceItem.tsx

```diff
@@ -12,76 +12,80 @@ import { TokenBalance } from 'src/tokens/slice'
 
 interface Props {
   token: TokenBalance
+  balanceUsdErrorFallback?: string
   onPress?: () => void
   containerStyle?: ViewStyle
 }
 
-export const TokenBalanceItem = ({ token, onPress, containerStyle }: Props) => {
+const TokenBalanceItemContent = ({ token, containerStyle, balanceUsdErrorFallback }: Props) => {
```

Comment:
nit: this extra component isn't really needed, we can store the JSX in a variable in the existing component.